.github/workflows/release_on_push.yml

```diff
@@ -26,6 +26,18 @@ jobs:
 
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
+      
+      - name: Download MiniLM ONNX model
+        run: |
+          mkdir -p mabl/src/main/assets
+          if [ ! -f mabl/src/main/assets/minilm-l6-v2-qint8-arm64.onnx ]; then
+            curl -L -o mabl/src/main/assets/minilm-l6-v2-qint8-arm64.onnx "https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/resolve/main/onnx/model_qint8_arm64.onnx?download=true"
+          fi
+
+      - name: Verify ONNX model present
```

Comment:
I don't think checking for the download is necessary. The previous step will already fail if `curl` returns a non-zero exit code.